### PR TITLE
Staking Competition Rules

### PR DESCRIPTION
**Pull request by ravenac95**

### CODE REVIEW

File: src/operators/staking-competition-rules.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
+# Rules and Guidelines for the Quest: Round 1
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]: "the Quest" vs "The Quest"?

File: src/operators/staking-competition-rules.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+1. Stress test the beta version of the Oasis Network before it Mainnets and give
+   feedback on what’s working - and what’s not.
+2. Give operators a chance to test drive the network and their setups.
+3. Give a diverse group of operators an opportunity to earn rewards that can be
+   used to stake on the Oasis Network Mainnet.
+4. Expand and further empower a community of data privacy advocates around the
+   Oasis Network.
+
+### Eligibility for the competition
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
### Eligibility for the Competition
```

File: src/operators/staking-competition-rules.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+We’ve received a number of questions about who can join, and we want to be as
+explicit as we can. We want to have as diverse and broad a community of node
+operators as possible. That said, citizens and residents of certain
+jurisdictions may not be eligible to receive rewards due to legal requirements.
+Your participation and eligibility to receive rewards are also subject to
+**KYC** and depending on your country of residence **accredited investor
+verification**. Before joining, be sure to read the official rules
+**[here][eligibility-rules]**.
+
+## How to join
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
## How to Join
```

File: src/operators/staking-competition-rules.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+operator will receive the same amount of test tokens to start the competition.
+
+5 days before the competition starts we’ll reach out to get your preferred
+public key. Please have this ready.
+
+We will let additional entities join after the start of the competition. 
+This will put you at a disadvantage for the most blocks signed reward but could
+still keep you in the running for best attack and most staked.
+
+## Types of challenges
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
## Types of Challenges
```

File: src/operators/staking-competition-rules.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+community that runs it -- are ready for a Mainnet launch. Our hope and plan is
+that the challenges we’ve set forward align to this goal as well. Challenges
+will vary in length and token rewards depending on what it is that is being tested. 
+Some more info below by length of challenge.
+
+Before diving into your strategy for all items listed below, we also encourage
+you to read through the [proposed incentive structure][incentives-proposal]
+for the Oasis Network.
+
+### Challenges spanning the full length of the competition
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
### Challenges Spanning the Entire Competition
```

File: src/operators/staking-competition-rules.md [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+
+  * Impact to other node operators
+  * Impact to network stability
+
+After the attack is complete, we ask you to write up your attack(s) (blog post,
+github repo, etc) and submit it to us on the #attacks channel on
+[slack][slack]. The Oasis team will review and choose the
+top 5 attacks based on the criteria above at the end of the competition.
+
+### Time-based competitions
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
### Time-based Challenges
```

File: src/operators/staking-competition-rules.md [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+Time-based competitions will focus on testing specific features and also
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
Time-based challenges will focus on testing specific features and also
```

File: src/operators/staking-competition-rules.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+  where all who upgrade in a given time period receive a token reward. Initially
+  we’ll let you know when we’re doing this and may also do a bit more
+  unannounced to keep everyone honest :).
+* **Delegation**: For one upgrade we’ll plan to turn delegation on. We’ll do
+  this after rewarding the most staked competition and will then ask people to
+  stress test our delegation mechanisms. More on the challenge later.
+* There may be more like the above. As they come we’ll share our plans and try
+  to give you ample time to prepare.
+
+### Other competitions / ways to win rewards
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
### Other Challenges / Ways to Win Rewards
```

File: src/operators/staking-competition-rules.md [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+   signed. We’ll share this via email, slack and on our blog.
+
+## Closing Ceremonies
+
+The length of the competition will largely depend on how stable the network
+proves to be, how active you all are, and what we learn about our incentive
+proposal during this competition. So while we can’t share an end date today, we
+can tell you that we’ll provide plenty of pre warning beforehand.
+
+## Resources to learn more
```
Comment (peterjgilbert) [inline src/operators/staking-competition-rules.md]:
```suggestion
## Resources to Learn More
```

File: src/operators/staking-competition-rules.md (hunk 1 above)
Comment (ravenac95) [reply src/operators/staking-competition-rules.md]: I like "The Quest"

Comment (ravenac95) [reply src/operators/staking-competition-rules.md]: @Afauvre ?

**Pull request merged**